plan/plan.go

```diff
@@ -224,27 +230,19 @@ type Module struct {
 type Account struct {
 	ComponentCommon `yaml:",inline"`
 
-	AccountBackends map[string]Backend      `yaml:"account_backends"` // Reference accounts for remote state
-	AllAccounts     map[string]*json.Number `yaml:"all_accounts"`
-	AccountName     string                  `yaml:"account_name"`
-	Global          *Component
+	Account string
+	Global  *Component
 }
 
 // Component is a component
 type Component struct {
 	ComponentCommon `yaml:",inline"`
 
-	Accounts        map[string]Account `yaml:"accounts"` // Reference accounts for remote state
-	AccountBackends map[string]Backend `yaml:"account_backends"`
-	Component       string             `yaml:"component"`
-	EKS             *v2.EKSConfig      `yaml:"eks,omitempty"`
-	Env             string
-
-	Kind              *v2.ComponentKind  `yaml:"kind,omitempty"`
-	ModuleSource      *string            `yaml:"module_source"`
-	ModuleName        *string            `yaml:"module_name"`
-	ComponentBackends map[string]Backend `yaml:"component_backends"`
-	Global            *Component         `yaml:"global"`
+	EKS          *v2.EKSConfig     `yaml:"eks,omitempty"`
```

Comment:
sidenote: I wonder if we can drop this since the AWS provider now has the ability to ignore tags